src/components/ProductsSidebar/index.jsx

```diff
@@ -0,0 +1,35 @@
+import { array, func } from 'prop-types'
+import React from 'react'
+
+export default function ProductsSidebar ({ categories = [], onHandleFilter, filters = [] }) {
+  const isActive = (id) => {
+    if (filters.filter((f) => f === id).length > 0) {
+      return 'filterIsActive'
+    }
+  }
```

Comment:
Every time this component `ProductsSidebar` renders the function `isActive` is assigned a new value, the function should be defined outside the definition of `ProductsSidebar` to prevent this extra work